sdk-setup/src/mb2

```diff
@@ -31,9 +31,9 @@ short_usage() {
     cat <<EOF
 usage: $ME [-t <target>] [-s <specfile>] [-d <device>] [-p <projectdir>]
            [-f <folder>] [-i] [-P] [-x]
-           build [-d] [-j <n>] [<args>] | qmake [<args>] | make [<args>] | ssh <args>
+           build [-e] [-d] [-j <n>] [<args>] | qmake [<args>] | make [<args>] | ssh <args>
```

Comment:
Because even though the flags do not overlap I didn't want to have -p for both global option (for projectdir) and dospec.. The long argument is still the same as with rpmbuild.